cli/src/params.rs

```diff
@@ -736,7 +736,31 @@ impl Parameters {
                 let key = typed_model.as_ref().unwrap().get_or_intern_symbol(key);
                 values.set(&key, value);
             }
-            stage!("set", typed_model -> typed_model, |m: TypedModel| {
+            stage!("set", typed_model -> typed_model, |mut m: TypedModel| {
+                for node in m.eval_order()? {
+                    let node = m.node_mut(node);
+                    if node.op_is::<Const>() {
+                        // safe to unwrap as we know it is a Const
+                        let op = node
+                            .op_as_mut::<Const>()
+                            .unwrap();
```

Comment:
I think you can simplify this a bit more using a `if let Some(op) = node.op_as::<Const> { ... }`